deltacat/storage/model/table_version.py

```diff
@@ -30,6 +32,7 @@ def of(
         table_version.description = description
         table_version.properties = properties
         table_version.content_types = content_types
+        table_version.sort_keys = sort_keys
```

Comment:
Could you explain what's the intention of adding sort_keys to table version?

Reply:
Table version has primary keys as the attribute, but not the sort key. It is not captured in the catalog object anywhere but used in compute module.